command/src/main/java/com/jagrosh/jdautilities/command/AnnotatedModuleCompiler.java

```diff
@@ -20,14 +20,14 @@
 
 /**
  * A "compiler" for {@link java.lang.Object Object}s that uses {@link java.lang.annotation.Annotation Annotation}s
- * as helpers for creating {@link com.jagrosh.jdautilities.command.Command Command}s.
+ * as helpers for creating {@link Command Command}s.
  *
  * <p>Previous to version 1.6 all Commands required the Command abstract class to be extended in source.
  * The primary issue that came with this was that Commands were restricted to that method of creation, offering
  * no support for popular means such as annotated commands.
  *
- * <p>Since 1.6 the introduction of {@link com.jagrosh.jdautilities.command.CommandBuilder CommandBuilder}
- * has allowed the potential to create unique {@link com.jagrosh.jdautilities.command.Command Command}
+ * <p>Since 1.6 the introduction of {@link CommandBuilder CommandBuilder}
+ * has allowed the potential to create unique {@link Command Command}
```

Comment:
This breaks JDA Documentation standards, the original way this was done should have been maintained